external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -220,14 +219,15 @@ void createDampening(RoutingContext routing, boolean isGroup) {
             @DocResponse(code = 400, message = "Bad Request/Invalid Parameters.", response = ResponseUtil.ApiError.class),
             @DocResponse(code = 500, message = "Internal server error.", response = ResponseUtil.ApiError.class)
     })
-    @Timed(absolute = true, tags = {"path=/triggers/trigger", "method=POST"})
+    @Timed
+            //(tags = {"path=/triggers/trigger", "method=POST"})
```

Comment:
Are the old tags automatically added by Micrometer? Should that comment be removed?